### ISSUE TITLE
Kanes Discount NYS - New Year's Sale with up to $500 in Free Furniture

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1263]
_Imported from trac ticket 1263.
Created by **kbower**
Opened in trac: Dec 1, 2011, 2:26:31 PM
Last modified in trac: Dec 28, 2011, 3:09:23 PM_
***
Kent,

Could you create a discount code NYS - New Year's Sale with up to $500 in Free Furniture with the standard exclusions that are also excluded from the plateau levels by 12/08/11 and have the following criteria:

a.$100 off total purchase of $999.00 to $1,998.99 or

b.$200 off total purchase of $1,999.00 to $3,998.99 or

c.$400 off total purchase of $3,999.00 to $4,998.99 or

d.$500 off total purchase of $4,999.00 or more

Thanks,

*Wayne Liburd*

*Director of MIS*

***
* wliburd@kanesfurniture.com attached [NYS - New Year's Sale with up to $500 in Free Furniture .xlsx](https://trac.retailarchitects.com/trac/attachment/ticket/1263/NYS - New Year's Sale with up to $500 in Free Furniture .xlsx) on Dec 7, 2011, 9:47:48 AM

**Comment by kbower** [comment]
_**kbower** trac comment on Dec 6, 2011, 12:11:47 PM:_

Discount ID:         NYS
Discount Description (40 characters max):         New Year's Sale with up to $500 in Free Furniture 
Discount applies to 'O'rder or 'I'tem (scope):         Item
If 'I'tem: Limited to a single item?         Single Item
Type: 'D'ollar or 'P'ercentage?         Dollar
Off retail price or line price?         Line Price
Discount stacks with what other discounts?         Ony with RAC discount code
Standard exclusions apply?         Yes
Any additional exclusions?        No
If discount amount is based on total purchase amount (plateaus)        
Do the exact same exclusions of an item being discounted also        
exclude that item from contributing to the total purchase amount?        Yes
Discount amount:        a.      $100 off total purchase of $999.00 to $1,998.99 or
        b.       $200 off total purchase of $1,999.00 to $3,998.99 or
        c.      $400 off total purchase of $3,999.00 to $4,998.99 or
        d.        $500 off total purchase of $4,999.00 or more
Needed for testing by:         12/8/2011

**Comment by kbower** [comment]
_**kbower** trac comment on Dec 6, 2011, 12:12:05 PM:_

Note: Description is 50 characters.

**Comment by kbower** [comment]
_**kbower** trac comment on Dec 7, 2011, 9:40:36 AM:_

Wayne,
    The description provided here is "$... off total purchase".  But in the spreadsheet you said it was an Item scope, limited to a single item.

Which one is it: "off total purchase" (Order scope) *or *off single item (limited by line price of the most expensive item)?

Kent

**Comment by kbower** [comment]
_**kbower** trac comment on Dec 7, 2011, 9:50:07 AM:_

_Updated:_
Discount applies to 'O'rder or 'I'tem (scope):         Order
  If 'I'tem: Limited to a single item?         N/A

**Comment by kbower** [comment]
_**kbower** trac comment on Dec 7, 2011, 10:02:43 AM:_

(Changeset [[2045]](../commit/97f1f734a43272f58bdff7905bdf5a77c48f878f))
 * Close #1263 (Kanes Discount NYS - New Year's Sale)

**Comment by kbower** [comment]
_**kbower** trac status change on Dec 7, 2011, 10:02:43 AM:_ **closed**

**Comment by kbower** [comment]
_**kbower** trac comment on Dec 7, 2011, 10:09:22 AM:_

# _Change [[2045]](../commit/97f1f734a43272f58bdff7905bdf5a77c48f878f) must be manually patched for LIVE! _
This line:
```
total_purchase = customerorder._kanes_discountpurchasetotal_c_
```must reference order instead.

**Comment by kbower** [comment]
_**wliburd@kanesfurniture.com** trac comment on Dec 28, 2011, 12:33:02 PM:_

Kent,

Good afternoon, there is a change in the plateau levels of the NYS sale which starts tomorrow 12/29/11. Could you make the following correction?

a.      $100 off total purchase of $1099.00 to $2,198.99 or

b.       $200 off total purchase of $2,199.00 to $4,398.99 or

c.      $400 off total purchase of $4,399.00 to $5,498.99 or

d.        $500 off total purchase of $5,499.00 or more

Thanks,

Wayne Liburd
Director of MIS
Kanes Furniture Corporation
5700 70th Ave North
Pinellas Park FL 33781
Tel: +1 (727) 545-9555 ext. 350
Fax: +1 (727) 541-6960
From: Kent Bower [mailto:kb@retailarchitects.com]
Sent: Wednesday, December 07, 2011 9:38 AM
To: Wayne Liburd
Subject: #1263: NYS - New Year's Sale with up to $500 in Free Furniture

Wayne,
    The description provided here is "$... off total purchase".  But in the spreadsheet you said it was an Item scope, limited to a single item.

Which one is it: "off total purchase" (Order scope) or off single item (limited by line price of the most expensive item)?

Kent



On 12/1/2011 11:44 AM, Wayne Liburd wrote:
Kent,

Could you create a discount code NYS - New Year's Sale with up to $500 in Free Furniture with the standard exclusions that are also excluded from the plateau levels by 12/08/11 and have the following criteria:

a.      $100 off total purchase of $999.00 to $1,998.99 or

b.       $200 off total purchase of $1,999.00 to $3,998.99 or

c.      $400 off total purchase of $3,999.00 to $4,998.99 or

d.        $500 off total purchase of $4,999.00 or more

Thanks,

Wayne Liburd
Director of MIS
Kanes Furniture Corporation
5700 70th Ave North
Pinellas Park FL 33781
Tel: +1 (727) 545-9555 ext. 350
Fax: +1 (727) 541-6960

**Comment by kbower** [comment]
_**kbower** trac comment on Dec 28, 2011, 2:54:30 PM:_

See above change needed.

**Comment by kbower** [comment]
_**kbower** trac status change on Dec 28, 2011, 2:54:30 PM:_ **reopened**

**Comment by kbower** [comment]
_**kbower** trac comment on Dec 28, 2011, 3:09:23 PM:_

(Changeset [[2085]](../commit/0cba3b4fdbeac91f5eb75edcdf8b024c0204aa2c))
 * adjust plateau levels of Kane's NYS sale (closes #1263)

**Comment by kbower** [comment]
_**kbower** trac status change on Dec 28, 2011, 3:09:23 PM:_ **closed**
